tests/acceptance/test_async.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import procrastinate
+
+pytestmark = pytest.mark.asyncio
```

Comment:
We (I :>) should look into doing this better :O There should be a fixture that lets us mark a test as `asyncio`. Then "asynchronous fixtures" like `async_app` could use this `asyncio` fixture. Thus we would not have to explicitly mark asynchronous tests.
The `db` fixture in `pytest_django` works like that. It's an empty shell that just adds the marker to the test.
(There's no call to action here, just the ramblings of a future old fool :santa: )

Reply:
Linked issue: https://github.com/peopledoc/procrastinate/issues/95